src/nft/components/explore/Explore.css.ts

```diff
@@ -204,7 +205,106 @@ export const valuePropContent = style([
         width: '58%',
         lineHeight: '36px',
       },
-      [`(min-width: ${breakpoints.lg}px)`]: { width: '50%' },
+      [`(min-width: ${breakpoints.lg - 1}px)`]: { width: '50%' },
+    },
+  },
+])
+
+/* Base Table Styles  */
+
+export const table = style([
+  {
+    borderCollapse: 'collapse',
+    borderStyle: 'hidden',
+    boxShadow: '0 0 0 1px rgba(153, 161, 189, 0.24)',
+    borderSpacing: '0px 40px',
+  },
+  sprinkles({
+    width: 'full',
+    borderRadius: '12',
+  }),
+])
+
+export const thead = style({
+  borderBottom: '1px solid rgba(153, 161, 189, 0.24)',
+  marginRight: '12px',
+})
+
+export const th = style([
+  {
+    selectors: {
+      '&:nth-last-child(1)': {
+        paddingRight: '20px',
+      },
+    },
+  },
+  sprinkles({
+    color: { default: 'textSecondary', hover: 'textPrimary' },
+    cursor: 'pointer',
+    paddingTop: '12',
+    paddingBottom: '12',
+    fontWeight: 'normal',
+    fontSize: '12',
+  }),
+])
+
+export const tr = sprinkles({ cursor: 'pointer' })
+
+export const rank = style([
+  sprinkles({
+    color: 'textSecondary',
+    position: 'absolute',
+    display: { md: 'inline-block', sm: 'none' },
+  }),
+  {
+    left: '24px',
+    top: '20px',
+  },
+])
+
+export const td = style([
+  body,
+  {
+    verticalAlign: 'middle',
+    selectors: {
+      '&:nth-last-child(1)': {
+        paddingRight: '20px',
+      },
     },
   },
+  sprinkles({
+    maxWidth: '160',
+    paddingTop: '10',
+    paddingBottom: '10',
+    textAlign: 'right',
+    position: 'relative',
+  }),
+])
+
+export const trendingOptions = style([
+  {
+    border: '2px solid rgba(153, 161, 189, 0.24)',
+  },
+  sprinkles({
+    marginBottom: '32',
+    height: '44',
+    borderRadius: '12',
+  }),
+])
+
+/* Trending Colletion styles */
+export const trendingOption = style([
+  {
+    padding: '14px 16px',
+    marginTop: '-1px',
+    marginLeft: '-1px',
+  },
+  sprinkles({
+    borderRadius: '12',
+    fontSize: '12',
+    display: 'inline-block',
+    cursor: 'pointer',
+  }),
 ])
+
+export const trendingOptionActive = sprinkles({ backgroundColor: 'grey700' })
```

Comment:
is this color grey700 is dark and light mode?

Reply:
Updated it. The light mode designs didn't exist when this was originally built.